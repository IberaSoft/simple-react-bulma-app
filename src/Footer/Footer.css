.footer {
    align-items: center;
    background: oldlace;
    display: flex;
    grid-area: footer;
    height: 50px;
    justify-content: center;
    width: 100%;
}