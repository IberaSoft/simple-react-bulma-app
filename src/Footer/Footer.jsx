import React from 'react';
//css
import './Footer.css';

const Footer = () => (
    <footer className="footer">
        Footer
    </footer>
)

export default Footer;