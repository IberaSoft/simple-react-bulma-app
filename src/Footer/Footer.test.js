import React from 'react'
import Footer from './Footer'
import renderer from 'react-test-renderer';

describe('Footer component', () => {
    it('Should match the snapshot', () => {
        const wrapper = renderer
            .create(<Footer />)
            .toJSON();
        expect(wrapper).toMatchSnapshot();
    });
})