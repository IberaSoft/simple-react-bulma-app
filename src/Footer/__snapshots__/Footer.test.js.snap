// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Footer component Should match the snapshot 1`] = `
<footer
  className="footer"
>
  <div
    className="content has-text-centered"
  >
    <p
      className="menu-label"
    >
      Footer
    </p>
  </div>
</footer>
`;
